src/book/01-team/01-introduction/03-current-team.mdx

```diff
@@ -26,15 +26,13 @@ slug: /team/current-members
 
 ### Senior Members
 
-These members have been on the team for a number of years and have attended at least one RoboCup competition.
+These members are an expert in at least one area of the team or systems.
 
-- **Daniel Ginn**: Currently studying for a Doctorate of Philosophy in Computer Science. Specialises in localisation.
+- **Kip Hamiltons**: Undergraduate student studying a combined degree in Mathematics and Computer Science. DevOps and Testing.
 
-- **Shawn Hain**: Undergraduate student studying a combined degree in Mechanical Engineering and Mechatronics Engineering.
+- **Cameron Murtagh**: Undergraduate student studying a combined degree in Mathematics and Computer Science. NUClear and networking.
```

Comment:
I'm not an "expert" on networking.

Reply:
Nor am I an expert on walk engines 😆 it's more what you do at NUbots. I can just make it 'NUClear'. Let me know if there's something else you'd prefer here.